tests/Unit/Elliptic/Triggers/Test_EveryNIterations.cpp

```diff
@@ -0,0 +1,49 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Elliptic/Triggers/EveryNIterations.hpp"
+#include "Parallel/PupStlCpp11.hpp"
+#include "Parallel/RegisterDerivedClassesWithCharm.hpp"
+#include "ParallelAlgorithms/EventsAndTriggers/Trigger.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+#include "tests/Unit/TestCreation.hpp"
+#include "tests/Unit/TestHelpers.hpp"
```

Comment:
```
#include <cstddef>
#include <initializer_list>
#include <string>
#include "DataStructures/DataBox/DataBoxTag.hpp"
```
(IWYU also wants `<vector>` and `"Utilities/Registration.hpp"`, but I don't see any uses of those.)